src/components/v2v3/shared/FundingCycleConfigurationDrawers/NftDrawer/index.tsx

```diff
@@ -8,9 +8,10 @@ import { useAppSelector } from 'hooks/AppSelector'
 
 import { useForm } from 'antd/lib/form/Form'
```

Comment:
**suggestion** I don't love how big this file has gotten. It's quite hard to read and follow. I'd suggest we can break it up by encapsulating some of the dispatch logic into their own hooks (in their own files), and potentially breaking up the template into multiple components.

Reply:
this can be **non-blocking** if you like, but we should consider it tech debt and come back to it

Reply:
Yeah you're right. There could be enough differences between the `new collection` and `editing a collection` version of this component to justify having two separate components. e.g. your comment and my reply here https://github.com/jbx-protocol/juice-interface/pull/2352#discussion_r1010063254